src/ontology/ro-edit.owl

```diff
@@ -5988,6 +5990,25 @@ AnnotationAssertion(rdfs:label obo:RO_0009005 "has primary substance added"@en)
 SubObjectPropertyOf(obo:RO_0009005 obo:RO_0009001)
 IrreflexiveObjectProperty(obo:RO_0009005)
 
+# Object Property: obo:RO_0009006 (assay measures characteristic)
+
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0009006 "A relation between an assay and a characteristic, in which the assay generates a data item which is a measure of a characteristic."@en)
```

Comment:
Could you add an example of usage, please?

Reply:
Done!